README.md

```diff
@@ -218,96 +370,136 @@ manually update all the previous data in the database.
 
 ### Transient Properties
 
-We can use the `@Transient` annotation if we don’t want a property to be used
-for database creation.
+We can use the `@Transient` annotation if we don’t want a property to be saved to the database.
+For example, if we were to add a `debugMessage` property (don't add it, this is just an example),
+the property would not be stored in the database table.
+
+```java
+    @Transient
+    private String debugMessage;
+
+```
+
+## Primary Key Generation
+
+The `@GeneratedValue` annotation makes the database generate unique IDs
```

Comment:
Line 393 in the code below has ```@Generated``` rather than ```@GeneratedValue```

Reply:
thanks, fixed.